.cirrus.yml

```diff
@@ -214,8 +214,8 @@ qa_task:
     cpu: 6
     memory: 16G
   env:
-    CLION_KEY: VAULT[development/team/sonarlint/kv/data/clion.key data.key]
-    GOLAND_KEY: VAULT[development/team/sonarlint/kv/data/goland.key data.key]
+    CLION_KEY: VAULT[development/team/sonarlint/kv/data/ide.keys clion]
+    GOLAND_KEY: VAULT[development/team/sonarlint/kv/data/ide.keys goland]
```

Comment:
it's data.goland
The KV secrets engine returns an array called data to cirrus, thus you need to get the key goland in the array data

Reply:
done